src/index.ios.js

```diff
@@ -0,0 +1,2 @@
+import DrawerLayout from 'react-native-drawer-layout';
```

Comment:
`export { default as default } from 'react-native-drawer-layout';` should require any plugin